osquery/tables/applications/browser_utils.cpp

```diff
@@ -137,10 +138,37 @@ void genExtension(const std::string& uid,
     permission_list = boost::algorithm::join(perm_vector, ", ");
   }
 
+  // Fetch the optional permission array from the manifest file
+  std::string optional_permission_list;
```

Comment:
Could we collapse the `permissions` and `optional_permissions` into a function?

Reply:
Yes definitely.